src/mame/includes/rltennis.h

```diff
@@ -1,38 +1,36 @@
 // license:BSD-3-Clause
 // copyright-holders:Tomasz Slanina
 #include "sound/dac.h"
-
+#include <algorithm>
 
 #define RLT_NUM_BLITTER_REGS    8
 #define RLT_NUM_BITMAPS         8
 
 class rltennis_state : public driver_device
 {
 public:
-	rltennis_state(const machine_config &mconfig, device_type type, const char *tag)
-		: driver_device(mconfig, type, tag),
+	rltennis_state(const machine_config &mconfig, device_type type, const char *tag) :
+		driver_device(mconfig, type, tag),
 		m_maincpu(*this, "maincpu"),
-		m_dac1(*this, "dac1"),
-		m_dac2(*this, "dac2"),
-		m_data760000(0), m_data740000(0), m_dac_counter(0), m_sample_rom_offset_1(0), m_sample_rom_offset_2(0),
-		m_offset_shift(0) { }
+		m_dac(*this, "dac%u", 1U),
+		m_samples(*this, "samples%u", 1U),
+		m_gfx(*this, "gfx"),
+		m_data760000(0), m_data740000(0), m_dac_counter(0),
+		m_offset_shift(0) { std::fill(std::begin(m_sample_rom_offset), std::end(m_sample_rom_offset), 0); }
 
```

Comment:
Don't put constructor body on the same line, more like this:
`
		m_blah1(x),
		m_blah2(x)
	{
		std::fill(std::begin(m_sample_rom_offset), std::end(m_sample_rom_offset), 0);
	}
`